### ISSUE TITLE
Private Chat Example?

### ISSUE THREAD

**Issue by markheramis** [feathersjs/feathers-chat#63]
Hello,

First of all, this is a great framework, and i like this example. But i'd be nice if a private chat is implemented with it (if possible). tnx.

**Comment by kuncevic** [comment]
Yeah it would of be nice to have. Unless it is not there may be some one can point out to the right direction? Currently the client listens to `/messages` so everyone who connected to that receive the list of messages and can send. 
As far as I can imagine to implement private messages the channel has to be setup between two users like `/messages/getUniqueChannelId(${user_one_id}, ${user_two_id})` so this way they both can listen to the same channel. 

The server should know how to create these channels at first. Say I am clicking on user name so the channel should be created and then If I am sending a message or same user send me a message we both should be kinda listen to the same channel. So here is a question how would you make another user to listen to the channel you just created?

**Comment by daffl** [comment]
There are some references for this in https://docs.feathersjs.com/faq/readme.html#how-do-i-create-channels-or-rooms. A private message is pretty much the same as a room/channel.

**Comment by kuncevic** [comment]
Just started to implement the rooms in https://github.com/kuncevic/feathers-chat/ currently just did one single commit https://github.com/kuncevic/feathers-chat/commit/1e9a21c244d04a0c803b1555fac93912bd438bba
So now when user is get signed up it is randomly gets assigned two rooms (from 1 to 3)
In Vanilla example I added new button `join` and input field where you can specify the `room_id` you want to join and also input field where you can specify the `room_id` you want to send a message to.

So the thing is that I just still figuring out the `function joinRoom` https://github.com/kuncevic/feathers-chat/blob/master/public/vanilla/app.js#L230 also this part https://github.com/kuncevic/feathers-chat/blob/master/src/services/messages/messages.service.js#L22 for now.

@daffl any thought/help on that?
